src/seed.ts

```diff
@@ -1,26 +1,31 @@
 import config from 'config';
 import { $ } from 'zx';
 import { createSeedYamlFile } from './yaml/creator';
+import { validateSupportedCache } from './common/validations';
 
 // eslint-disable-next-line @typescript-eslint/consistent-type-definitions
 export type SeedOptions = {
   cache: string;
+  grid: string;
   fromZoomLevel: number;
   toZoomLevel: number;
   refreshBefore: string;
   wktFilePath: string;
   skipUncached: boolean;
+  concurrency: number;
 };
+
 export const executeSeed = async (options: SeedOptions): Promise<void> => {
   const mapproxyYamlFilePath = config.get<string>('mapproxyYamlFilePath');
   const seedYamlFilePath = config.get<string>('seedYamlFilePath');
   const flags: string[] = [];
 
+  validateSupportedCache(mapproxyYamlFilePath, options.cache);
```

Comment:
add console before validating
add console before seed yaml creation
add console before running the seed